service/src/main/java/bio/terra/axonserver/service/convert/ConvertService.java

```diff
@@ -22,17 +24,14 @@ public ConvertService(CalhounService calhounService) {
    * expected convertTo format.
    *
    * @param file The file to convert
-   * @param fileExtension The extension of the file to convert
    * @param convertTo The format to convert the file to
    * @param token Bearer token
    * @return The converted file
    * @throws InvalidConvertToFormat If the convertTo format is not supported
    */
-  public byte[] convertFile(
-      byte[] file, String fileExtension, String convertTo, BearerToken token) {
-
+  public File convertFile(File file, String convertTo, BearerToken token) {
     if (convertTo.equalsIgnoreCase("html")) {
```

Comment:
Do we ever (or plan to ever) convert to non-html formats? if not, it seems like you could just remove all of the `convertTo` parameters and checks

Reply:
`convertTo` was orignally specced in the axon server design doc. I'm not aware of any formats we'll need in the short term, but perhaps there will clearer requirements with the app framework work. Thoughts on keeping as is for now?